cupyx/linalg/solve.py

```diff
@@ -7,6 +7,75 @@
 from cupy.linalg import util
 
 
+def _batched_invh(a):
```

Comment:
To obtain the inverse matrix we solve the linear system with the right-hand-side as an identity matrix, so in this case, we always have `nrhs` > 1 (unless for a trivial case of 1x1 matrix).
I have added an xfailing test, once Nvidia fixes the problem with `potrsBatched` it will be noticed with the tests.

Reply:
You are right.  Now I understand what cuSOLVER supports corresponds to "batched" `scipy.linalg.solve(a, b, assume_a='sym')` (`a.shape == (..., n, n)` and `b.shape == (..., n, nhrs)` with `nhrs == 1`).